instaclone/src/components/CommentSection/CommetSection.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import './comment-section.css';
+
+class CommentSection extends React.Component {
+    constructor(props){
+        super(props);
+        this.state={
+            comments: this.props.element.comments
+        }
+                        }
+
+       handleChanges = e => {
+        this.setState({
+          [e.target.name]: e.target.value
+        })
+      }
+    
+      addComment = e => { 
+        e.preventDefault();
+        const newComment = {
+          username: "",
```

Comment:
You could store the username in localStorage when the user logs in and the use it for the comments